torchdynamo/symbolic_convert.py

```diff
@@ -1225,6 +1225,17 @@ def __init__(
             symbolic_globals=collections.OrderedDict(),
             f_code=f_code,
         )
+
+        self.new_annotations = {}
+
+        if "self" in f_locals:
```

Comment:
I actually liked your previous trick a bit better.  (The one using `gc.get_referrers`.)

I hadn't thought of it when I suggested this one.  This one is worse.